tests/release/e2e-test-push-image-to-pyxis.go

```diff
@@ -147,68 +159,128 @@ var _ = framework.ReleaseSuiteDescribe("[HACBS-1571]test-release-e2e-push-image-
 
 	var _ = Describe("Post-release verification", func() {
 
-		It("verifies that a PipelineRun is created in dev namespace.", func() {
+		It("verifies that a PipelineRuns is created in dev namespace.", func() {
 			Eventually(func() bool {
 				prList, err := fw.AsKubeAdmin.TektonController.ListAllPipelineRuns(devNamespace)
 				if err != nil || prList == nil || len(prList.Items) < 1 {
 					return false
 				}
+				componentHasBuildPr := false
+				additionalComponentHasBuildPr := false
+				for _, pr := range prList.Items {
+					if strings.Contains(pr.Name, componentName) {
+						componentHasBuildPr = true
+						buildPrName = pr.Name
+					} else if strings.Contains(pr.Name, additionalComponentName) {
+						additionalComponentHasBuildPr = true
+						additionalBuildPrName = pr.Name
+					}
+				}
+
+				GinkgoWriter.Printf("\nFirst Build PirpelineRun:\n %s", buildPrName)
+				GinkgoWriter.Printf("\nSecond Build PirpelineRun:\n %s", additionalBuildPrName)
 
-				return strings.Contains(prList.Items[0].Name, componentName)
+				return componentHasBuildPr && additionalComponentHasBuildPr
 			}, releasePipelineRunCreationTimeout, defaultInterval).Should(BeTrue())
 		})
 
-		It("verifies that the PipelineRun in dev namespace succeeded.", func() {
+		It("verifies that the build PipelineRuns in dev namespace succeeded.", func() {
 			Eventually(func() bool {
-				prList, err := fw.AsKubeAdmin.TektonController.ListAllPipelineRuns(devNamespace)
-				if prList == nil || err != nil || len(prList.Items) < 1 {
+				buildPr, err := fw.AsKubeAdmin.TektonController.GetPipelineRun(buildPrName, devNamespace)
+				if err != nil {
+					GinkgoWriter.Printf("\nError getting PirpelineRun %s:\n %s", buildPrName, err)
+					return false
+				}
+				additionalBuildPr, err := fw.AsKubeAdmin.TektonController.GetPipelineRun(additionalBuildPrName, devNamespace)
+				if err != nil {
+					GinkgoWriter.Printf("\nError getting PirpelineRun %s:\n %s", additionalBuildPr, err)
```

Comment:
Typo: -> `PipelineRun`

Reply:
There are actually a number of other occurrences of `PirpelineRun` - you can search and replace.